### ISSUE TITLE
Metrics

### ISSUE THREAD

**Issue by djanibekov** [vs-sat-dev/Body-performance-Aliftech-test-task-#1]
@vs-sat-dev 
Please try to use some other metrics, particularly for classification problems the "confusion matrix" is one of the most important metric

`from sklearn.metrics import confusion_matrix`
 `from sklearn.metrics import classification_report`

**Comment by vs-sat-dev** [comment]
Thank you. Confusion matrix was in my plains.

**Comment by djanibekov** [comment]
One more question about the way of using those metrics. Why did not you used them after `optuna` fine-tuning?
Firstly, you applied them to see how well your default model is performing. Usually, default model metrics are not good enough for MVP (Minimum valuable product). However, in your case the fine tuning process did not get any improvement. So , it is OK in this case

Additionally, accuracy is not best suited metric for this problem, namely for multi class classification, to optimize due to imbalanced distributions. For example, if there is a severe imbalances a model will constantly predict the majority of classes

This is just a nagging😅.

**Comment by vs-sat-dev** [comment]
Hi. Thank you for your feedback. I used confusion_matrix after optuna tuning with different metrics but results were worse or the same. Yesterday I read a lot about metrics for multiclass prediction and got a little better understanding of it. I think accuracy was good enough for this dataset it was comparable with the f1-score.
